.travis.yml

```diff
@@ -1,16 +1,47 @@
 language: python
-python: 2.7
+python: 
+  - 2.7
+  - 3.5
 addons:
   firefox: 47.0.1
-env:
-- TOXENV=tests DISPLAY=:99.0
-- TOXENV=flake8
+matrix:
+  include:
+    - python: 2.7
+      env: 
+        - TOXENV=flake8
+    - python: 2.7
+      env: 
+        - DISPLAY=:99.0
+        - TOXENV=py27
+    - python: 2.7
+      env: 
+        - DISPLAY=:99.0
+        - TOXENV=py27-marionette
+    - python: 3.5
+      env: TOXENV=flake8
+    - python: 3.5
+      env: 
+        - DISPLAY=:99.0
+        - TOXENV=py35
```

Comment:
This entry isn't needed, as `tox-travis` will discover this environment based on the python version in use.